keyboards/purin/readme.md

```diff
@@ -0,0 +1,28 @@
+# purin
+
+![purin](https://i.imgur.com/ig8DWEF.jpg)
```

Comment:
```suggestion
![purin](https://i.imgur.com/ig8DWEFh.jpg)
```

Reply:
Thank you! but I have a small question (using the imgur for the first time), what is the difference between using h or not?

Reply:
The h resizes the image

For reference:
```
s =   90×  90 = Small Square (as seen in the example above)
b =  160× 160 = Big Square 
t =  160× 160 = Small Thumbnail 
m =  320× 320 = Medium Thumbnail 
l =  640× 640 = Large Thumbnail
h = 1024×1024 = Huge Thumbnail
```